decidim-core/config/locales/en.yml

```diff
@@ -87,6 +87,7 @@ en:
       decidim_with_month_name: "%d %B %Y"
       decidim_with_month_name_short: "%d %b"
   datetime:
+    ago: "%{time} ago"
```

Comment:
This is again one of those where it is a bit hard to draw the line between a fix and a change. We should avoid removing translations or adding new ones when we do backports while fixing clear translation mistakes should be generally OK as fixes.

But wouldn't it be possible to do using the existing translation key, even if it does not perfectly describe the context where it is used?